components/icons/navbar_icons.tsx

```diff
@@ -0,0 +1,119 @@
+      fill="white"
+    />
+    <path
+      d="M77.1474 22.7928C76.0844 22.7928 75.3485 23.3576 75.3485 24.1241C75.3485 24.9713 76.3297 25.294 76.9021 25.4554L77.7198 25.6571C78.5783 25.8588 80.0093 26.3429 80.0093 27.8759C80.0093 29.1668 78.9463 30.1753 77.0656 30.1753C75.3076 30.1753 74.2447 29.2878 74.1629 27.9565H75.185C75.2668 28.8844 76.1253 29.2878 77.1065 29.2878C78.2513 29.2878 79.1098 28.723 79.1098 27.8355C79.1098 27.0287 78.333 26.7463 77.4745 26.5042L76.4933 26.2219C75.2259 25.8588 74.4491 25.2133 74.4491 24.1241C74.4491 22.7928 75.6756 21.865 77.2292 21.865C78.7828 21.865 79.9275 22.7525 79.9684 23.9627H79.0281C78.8645 23.2366 78.1286 22.7928 77.1474 22.7928Z"
+      fill="white"
+    />
+    <path
+      d="M80.9905 22.0264H87.0821V22.8735H84.5473V30.014H83.5661V22.8735H81.0314L80.9905 22.0264Z"
+      fill="white"
+    />
+    <path d="M89.4534 30.0543H88.4722V22.0264H89.4534V30.0543Z" fill="white" />
+    <path
+      d="M97.8754 30.0543H96.9351L92.5196 23.761H92.4379V30.0543H91.4567V22.0264H92.397L96.8533 28.3196H96.9351V22.0264H97.9163L97.8754 30.0543Z"
+      fill="white"
+    />
+    <path
+      d="M103.068 22.8332C101.637 22.8332 100.451 23.9627 100.451 26.0605C100.451 28.1582 101.596 29.2878 103.108 29.2878C104.499 29.2878 105.439 28.36 105.48 26.948H103.354V26.1008H106.461V26.948C106.461 28.9247 105.071 30.2157 103.149 30.2157C100.983 30.2157 99.5107 28.6423 99.5107 26.1008C99.5107 23.5593 100.983 21.986 103.068 21.986C104.785 21.986 106.093 23.0349 106.42 24.6082H105.398C105.03 23.4787 104.253 22.8332 103.068 22.8332Z"
+      fill="white"
+    />
+    <path
+      d="M35.3234 34.6532C40.1886 34.6532 44.1134 30.7805 44.1134 25.9798C44.1134 21.1792 40.1886 17.3065 35.3234 17.3065C30.4583 17.3065 26.5334 21.1792 26.5334 25.9798C26.5334 30.7805 30.4583 34.6532 35.3234 34.6532Z"
+      fill="#FFC629"
+    />
+    <g clip-path="url(#clip0_209_853)">
+      <path
+        fill-rule="evenodd"
+        clip-rule="evenodd"
+        d="M88.2498 36.0702C88.3908 36.2094 88.3908 36.4351 88.2498 36.5743L85.2448 39.5408L88.1672 42.4257C88.3082 42.5649 88.3082 42.7907 88.1672 42.9299C88.0262 43.0691 87.7976 43.0691 87.6565 42.9299L84.4788 39.7928C84.411 39.726 84.373 39.6353 84.373 39.5408C84.373 39.4462 84.411 39.3555 84.4788 39.2887L87.7391 36.0702C87.8801 35.931 88.1087 35.931 88.2498 36.0702Z"
+        fill="white"
+      />
+      <path
+        fill-rule="evenodd"
+        clip-rule="evenodd"
+        d="M98.7736 36.0702C98.9146 36.2094 98.9146 36.4351 98.7736 36.5743L95.7686 39.5408L98.691 42.4257C98.832 42.5649 98.832 42.7907 98.691 42.9299C98.55 43.0691 98.3214 43.0691 98.1803 42.9299L95.0026 39.7928C94.9348 39.726 94.8968 39.6353 94.8968 39.5408C94.8968 39.4462 94.9348 39.3555 95.0026 39.2887L98.2629 36.0702C98.4039 35.931 98.6325 35.931 98.7736 36.0702Z"
+        fill="white"
+      />
+      <path
```

Comment:
ok this is all pretty messy—we'll have a cleaner Nowcasting logo component once #42 is merged in. this is fine for now, but we'll have to remember to change this once that gets merged.